### ISSUE TITLE
enable double tap action on Apple Pencil when tool picker is not shown

### ISSUE THREAD

**Issue by 0si43** [0si43/LikePaper#24]
Now, double tap action on Apple Pencil work when tool picker is shown.
When tool picker is out of sight, double tap action is ignored.
It's not critical, but a little inconvenient.

**Comment by 0si43** [comment]
[Handling Double Taps from Apple Pencil](https://developer.apple.com/documentation/uikit/pencil_interactions/handling_double_taps_from_apple_pencil)

**Comment by 0si43** [comment]
When PKToolPicker is shown, double tap action is enabled.
But when PKToolPicker is hidden, it's ignored.
For becoming double tap action enabled without visible PKTooPicker, I need to add some codes handling `UIPencilInteraction `.

**Comment by 0si43** [comment]
By testing my code, I was confused double tap action is disabled when PKToolPicker is visible.

I noticed `toolPickerSelectedToolDidChange` called twice.
Why?
I added double-tap action manually, and PKToolPicker do same action!
So, I switched InkTool to eraseTool, then PKToolPicker switched eraseTool to InkTool.
The result is seemed nothing happen for me.

```swift
func pencilInteractionDidTap(_ interaction: UIPencilInteraction) {
        guard !toolPicker.isVisible else { return }
    }
```
I inserted guard-section, so it's work!
